include/rppt_tensor_effects_augmentations.h

```diff
@@ -685,6 +685,52 @@ RppStatus rppt_pixelate_host(RppPtr_t srcPtr, RpptDescPtr srcDescPtr, RppPtr_t d
 RppStatus rppt_pixelate_gpu(RppPtr_t srcPtr, RpptDescPtr srcDescPtr, RppPtr_t dstPtr, RpptDescPtr dstDescPtr, RppPtr_t intermediateScratchBufferPtr, Rpp32f pixelationPercentage, RpptROIPtr roiTensorPtrSrc, RpptRoiType roiType, rppHandle_t rppHandle);
 #endif // GPU_SUPPORT
 
+/*! \brief Snow augmentation on HOST backend for a NCHW/NHWC layout tensor
+ * \details The Snow augmentation does a  modification of brightness on a batch of RGB(3 channel) / greyscale(1 channel) images with an NHWC/NCHW tensor layout.<br>
+ * - srcPtr depth ranges - Rpp8u (0 to 255), Rpp16f (0 to 1), Rpp32f (0 to 1), Rpp8s (-128 to 127).
+ * - dstPtr depth ranges - Will be same depth as srcPtr.
+ * \image html img640x480.png Sample Input
```

Comment:
Same doubt here

Reply:
We used this dashcam image  for fog PR . https://github.com/r-abishek/rpp/pull/332/files#diff-006ea80a28a2d71eeb553a7a9c8b32912f4d35f871150a5094fbe85e3503575f . So I used same .